NetKAN/RealBattery.netkan

```diff
@@ -0,0 +1,16 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "RealBattery",
+    "$kref"        : "#/ckan/github/blackliner/RealBattery",
+    "license"      : "MIT",
+    "ksp_version"  : "1.2",
```

Comment:
thanks for the hint. I knew that exists, but i was not shure if it works 100%. Ill try it :-)